docs/psII_tutorial.md

```diff
@@ -110,15 +110,16 @@ def main():
 
 ```
 
-**Figure 4.** Rotated `fmax` frame  
+**Figure 4.** Rotated `fmin` frame  
 
-![Screenshot](img/tutorial_images/psII/fmax_rescaled.jpg)
+![Screenshot](img/tutorial_images/psII/fmin2.png)
 
 The resulting image is then thresholded with a [binary threshold](binary_threshold.md) to capture the plant material. In most cases, it is expected that pixel values
 range between 0 and 255, but our example image has pixel values from 0 to over 7000. Trial and error is a common method for selecting an appropriate threshold value.
+We recommend choosing a threshold based on the Fmin image to reduce noise that may be introduced by the bright imaging light used when capturing Fmax. If this is not possible, we recommend using the [erode](erode.md) function after the [fill](fill.md) step to reduce noise around the plant border.
```

Comment:
I think it would be better to say something like:
... and in different imaging scenarios it may be easier to segment Fmin or Fmax. Fmax will have higher dynamic range to more easily pick a precise threshold value but Fmin may have less errant signal, e.g. if the leaf was moving during imaging.

Reply:
Yeah, I like that wording better. I think it makes the options clearer. I'll update the text.